lib/ui/state/hide_balance_state.dart

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: 2022 Foundation Devices Inc.
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import 'dart:convert';
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../business/account.dart';
```

Comment:
Can we use `package:` style imports instead of relative?

Reply:
got it